ChemoTracker/components/PatientEducation/Search.js

```diff
@@ -0,0 +1,173 @@
+import React, { Component } from 'react';
+import { ActivityIndicator, FlatList, Linking, ListView, Text, TextInput, TouchableOpacity, View } from 'react-native';
+import { List, ListItem } from "react-native-elements";
+import { TabNavigator } from 'react-navigation';
+import { WebBrowser } from 'expo';
+import FontAwesomeIcon from 'react-native-vector-icons/FontAwesome';
+import IonIcon from 'react-native-vector-icons/Ionicons';
+import { searchStyles } from '../../styles/PatientEducation/patient-education.js';
+import color from '../../styles/color.js';
+
+const api = 'http://ec2-52-15-106-40.us-east-2.compute.amazonaws.com:8000';
+const header = {
+  'Accept': 'application/json',
+  'Content-Type': 'application/json'
+}
+
+export const getSearchResults = (searchQuery) => {
+  const queryparams = `query=${searchQuery}`;
+  return fetch(`${api}/patient-education/posts?${queryparams}`, {
+    method: 'get',
+    headers: {
```

Comment:
lol what's the use of line 12 if you are not using it here
i.e `headers: header`

Reply:
removing it 😄